kv/memberlist/tcp_transport_test.go

```diff
@@ -95,3 +95,10 @@ func TestFinalAdvertiseAddr(t *testing.T) {
 		})
 	}
 }
+
+func TestNonIPsAreRejected(t *testing.T) {
+	cfg := TCPTransportConfig{BindAddrs: flagext.StringSlice{"localhost"}}
+	_, err := NewTCPTransport(cfg, nil, nil)
+	require.Error(t, err)
```

Comment:
This `require.Error()` call is no longer necessary - `EqualError` will fail if `err` is `nil`.

Reply:
Ahhh, yes. This should be fixed now